### ISSUE TITLE
/etc/cron.daily/netdata-updater:  netdata: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by netdata)

### ISSUE THREAD

**Issue by AGI-chandler** [netdata/netdata#11890]
##### Bug report summary

I've been receiving the error since 12/2

##### OS / Environment
```
Linux EagI 4.18.0-305.25.1.el8_4.x86_64 #1 SMP Wed Nov 3 10:29:07 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
/etc/centos-release:CentOS Linux release 8.4.2105
```

##### Netdata version
Version: netdata v1.32.0
Configure options:  '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libexecdir=/usr/libexec' '--libdir=/usr/lib' '--with-zlib' '--with-math' '--with-user=netdata' '--with-bundled-lws' '--with-bundled-protobuf' 'CFLAGS=-O2' 'LDFLAGS='
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES 
    ACLK Next Generation:       YES
    ACLK-NG New Cloud Protocol: YES
    ACLK Legacy:                YES
    TLS Host Verification:      YES
    Machine Learning:           YES
Libraries:
    protobuf:                YES (bundled)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    LWS:                     YES static v3.2.2
    mosquitto:               YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    NO
    EBPF:                    YES
    IPMI:                    NO
    NFACCT:                  NO
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: NO

##### Installation method
I used the one-line installation script with --stable-channel

##### Component Name
netdata-updater

##### Steps To Reproduce

1. I guess you could install netdata with the one-line script using --stable-channel, but I've had it installed for several months and it was working fine so I doubt this would work.  Like I said the error started showing up on 12/2, maybe it was an update I installed on the system?  I haven't updated anything for about a month though (yum update).  I guess it is a problem with the updater trying to install a newer version but it can't find the required glibcxx? which is weird since it's the latest version of CentOS....

##### Expected behavior

netdata-updater runs successfully

**Comment by ilyam8** [comment]
Hey, @AGI-chandler. Are you still experiencing this issue? Have you updated to v1.32.1?

**Comment by ilyam8** [comment]
> downloading netdata-v1.32.1.gz.run

That is [a static binary package](https://learn.netdata.cloud/docs/agent/packaging/installer/methods/kickstart-64), it installs Netdata at `/opt/netdata/`. You are using other method ([kickstart.sh](https://learn.netdata.cloud/docs/agent/packaging/installer/methods/kickstart)).


> Wed Dec 22 05:46:23 MST 2021 : INFO:  Current Version: 00103200100000
> Wed Dec 22 05:46:23 MST 2021 : INFO:  Latest Version: 00103200100000

As we see you have been successfully updated to the latest stable release (v1.32.1 atm).

**Comment by ilyam8** [comment]
> I've been receiving the error since 12/2

Does it still occur?

**Comment by AGI-chandler** [comment]
Yes I'm still getting the error each day in my email, last received today at 04:18

**Comment by MrZammler** [comment]
Hi @AGI-chandler !

Can you please paste the output of `strings /lib64/libstdc++.so.6 | grep GLIBC` and `objdump -T /usr/sbin/netdata | grep GLIBCXX_3.4.26` ?

**Comment by AGI-chandler** [comment]
Sure @MrZammler :

```
$ strings /lib64/libstdc++.so.6 | grep GLIBC
GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBCXX_3.4.14
GLIBCXX_3.4.15
GLIBCXX_3.4.16
GLIBCXX_3.4.17
GLIBCXX_3.4.18
GLIBCXX_3.4.19
GLIBCXX_3.4.20
GLIBCXX_3.4.21
GLIBCXX_3.4.22
GLIBCXX_3.4.23
GLIBCXX_3.4.24
GLIBCXX_3.4.25
GLIBC_2.2.5
GLIBC_2.3
GLIBC_2.14
GLIBC_2.4
GLIBC_2.18
GLIBC_2.16
GLIBC_2.17
GLIBC_2.3.2
GLIBCXX_DEBUG_MESSAGE_LENGTH
GA+GLIBCXX_ASSERTIONS

$ objdump -T /usr/sbin/netdata | grep GLIBCXX_3.4.26
0000000000000000      DF *UND*	0000000000000000  GLIBCXX_3.4.26 _ZNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEEC1Ev

$ 
```

**Comment by MrZammler** [comment]
Hi @AGI-chandler sorry for my very late reply...

Can you please check that your `libstdc++6` is up to date? Thanks, and sorry again!

**Comment by AGI-chandler** [comment]
Yes I updated libstdc++ package but it didn't help, the strings and objdump output are the same as above.

**Comment by cpipilas** [comment]
Closing this issue due to inactivity, update to the latest netdata version and let us know if the issue is still valid.
